src/popup/commentCounter.js

```diff
@@ -0,0 +1,7 @@
+export const countComments = async (comments) => {
+  const commentCounter = document.querySelector('.counter');
+  const displayCount = await comments.length;
+  comments.forEach(() => {
+    commentCounter.innerHTML = `(${displayCount})`;
+  });
+};
```

Comment:
The comment counter is currently not implemented as expected, here you are expected to implement it just as you did for the item counter by searching the Dom for a particular element and then perform the calculation. Kindly address this to stay in sync with the project requirements.

NOTE: Please let this cover for when there are no comments at all on the particular item